helper_dart/pubspec.yaml

```diff
@@ -6,12 +6,12 @@ environment:
   sdk: '>=2.17.5 <3.0.0'
 dependencies:
   algolia: ^1.1.1
+  algolia_insights:
+    path: ../insights_dart
```

Comment:
When releasing, this line will be an issue :D

_Suggestion: maybe it's time to move to something such as [melos](https://pub.dev/packages/melos) for package managing!_